trace/opencensus/trace/config_integration.py

```diff
@@ -17,7 +17,7 @@
 
 log = logging.getLogger(__name__)
 
-SUPPORTED_INTEGRATIONS = ['mysql', 'postgresql']
+SUPPORTED_INTEGRATIONS = ['mysql', 'postgresql', 'sqlalchemy']
```

Comment:
What happens if I enable sqlalchemy and mysql or postgres integration at the same time?  Does everything get traced twice?  Should we add documentation and a warning for this case?

Reply:
The communication between SQLAlchemy and the database it connected will also be traced, and yes the query will be traced twice. Documentation added for this case.